modules/apps/transfer/keeper/grpc_query.go

```diff
@@ -134,9 +134,10 @@ func (k Keeper) TotalEscrowForDenom(c context.Context, req *types.QueryTotalEscr
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
-	denomAmount := k.GetTotalEscrowForDenom(ctx, req.Denom)
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.TotalEscrowForDenomKey(req.Denom))
 
 	return &types.QueryTotalEscrowForDenomResponse{
-		Amount: denomAmount,
```

Comment:
If we want to return a `string`, we can just do `denomAmount.String()`, right?

Reply:
yeah this would be a bit nicer, I will play with this idea